io.github.aganzha.Stage.desktop

```diff
@@ -0,0 +1,17 @@
+[Desktop Entry]
+Name=Stage
+Comment=GUI Git client inspited by Magit
+
+Keywords=GNOME;GTK;Git;Commt;Branch;Push;Pull;Merge;Rebase;Cherry-pick;Revert;
+Categories=Development;Utility;RevisionControl;Monitor;FileTools;GNOME;GTK;
```

Comment:
some of these don't seem like valid categories. You can't pick random things, it has a spec https://specifications.freedesktop.org/menu-spec/latest/category-registry.html

Reply:
Got it, thank you!